packages/vanilla/src/components/accordion/accordion.js

```diff
@@ -1,13 +1,25 @@
 class Accordion {
-  constructor(domNode) {
-    this.accordionNode = domNode;
+  constructor(accordionItem) {
+    this.accordionNode = accordionItem;
+    this.accordionTrigger = this.accordionNode.querySelector('.cbp-accordion__trigger');
```

Comment:
General comments on naming for consistency:

* Why not just "accordion" for the self-referencing DOM node? I think we've done that in others.
* Can we use the word "control" (instead of "trigger") for consistency, e.g., `accordionControl`?

Reply:
Changed names in accordion js.